Corgibytes.Freshli.Cli/CommandRunners/LoadServiceCommandRunner.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.CommandLine.Invocation;
+using Corgibytes.Freshli.Cli.CommandOptions;
+using Corgibytes.Freshli.Cli.Commands;
+using Corgibytes.Freshli.Cli.Functionality;
+using Corgibytes.Freshli.Cli.Functionality.Engine;
+using Corgibytes.Freshli.Lib;
+
+namespace Corgibytes.Freshli.Cli.CommandRunners;
+
+public class LoadServiceCommandRunner : CommandRunner<LoadServiceCommand, EmptyCommandOptions>
+{
+    public LoadServiceCommandRunner(IServiceProvider serviceProvider, Runner runner,
+        IApplicationActivityEngine activityEngine)
+        : base(serviceProvider, runner) =>
+        ActivityEngine = activityEngine;
+
+    private IApplicationActivityEngine ActivityEngine { get; }
+
+    public override int Run(EmptyCommandOptions options, InvocationContext context)
+    {
+        ActivityEngine.Dispatch(new LoadServiceProviderActivity());
+        ActivityEngine.Wait();
+        return 0;
+    }
+}
```

Comment:
I really like using this to test that services are getting loaded correctly. Nicely done!

Reply:
Thank you! :D